### DISCUSSION TITLE
Injection Site?

### DISCUSSION THREAD

**Discussion by Aran55336** [chaos-polymtl/lethe#892] [Q&A]
Is simulating a sparger/injection site of a different fluid possible? I have seen them in fluidized beds however I want to implement them at different heights of a tank, not necessarily at the very bottom.

**Comment by blaisb** [comment]
This has never been tested yet, but right now the CFD-DEm solver does not support multi-fluid on the CFD side.

**Reply by Aran55336** [reply]
What about just the CFD solver? Could I inject gas into a liquid at a height not necessarily at a boundary of the geometry?

**Reply by blaisb** [reply]
Dear Aran,
This would required a VOF volumetric source term. Could be done, it is not fully supported yet, but you would just need to implement a source term in the VOF equations.
